### PR TITLE
update to vanilla accordion pattern

### LINKED ISSUE #21: Use accordion pattern instead of current sidenav
On the following pages:
- /examples
- /downloads
- /brand

Use the MAAS/dashboard theme [accordion](https://github.com/ubuntudesign/vanilla-dashboard-theme-design/tree/master/Accordion) pattern

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* update to vanilla accordion pattern on downloads and brand pages

## QA

* download the branch
* `./run`
* look at [brand section](http://0.0.0.0:8011/brand/) and the [download page](http://0.0.0.0:8011/download/)

_note, there are some javascript issues on /download, but we will fix that in another PR_

## Issue / Card

* Fixes #21 

## Screenshots

image

### CODE REVIEW

File: _sass/_scratch.scss [hunk 1]
```diff
@@ -12,105 +12,9 @@
...
-    height: 8px;
-
-    background: url('https://assets.ubuntu.com/v1/37927a91-icon-accordion-inactive.png') 0 0 no-repeat transparent;
-}
-.ui-accordion .ui-state-active .ui-icon {
-    background-image: url('https://assets.ubuntu.com/v1/b281bfba-icon-accordion-active.png');
-}
-.ui-accordion .current_page_item a {
-    color: #333;
+.p-accordion__panel > p > label {
```
Comment (nottrobin) [inline _sass/_scratch.scss]: If there's a problem with the accordion pattern as it is provided by Vanilla, has it been filed as an issue?

File: brand/canonical-logo/index.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ body-class: "guidelines"
 ---
 <div class="p-strip">
   <div class="row">
-    <div id="loop-guidelines" class="col-10">
+    <div id="loop-guidelines" class="col-12">
```
Comment (nottrobin) [inline brand/canonical-logo/index.md]: Why? This seems like it's inconsistent with the rest of the page.

File: _includes/side-nav-scopes.html [hunk 3]
```diff
@@ -1,10 +0,0 @@
-
```
Comment (nottrobin) [inline _includes/side-nav-scopes.html]: This file is mentioned in `_layouts/post-with-banner.html` - but AFAICT that's not mentioned anywhere. If you're going to remove this one, you should presumably remove that one too.

File: _sass/_scratch.scss (hunk 1 above)
Comment (pmahnke) [reply _sass/_scratch.scss]: 1. the minus sign doesn't work in the [vanilla example](https://docs.vanillaframework.io/en/patterns/accordion) either; however, that is there as an example only, so I think I need help fixing it... I looked and don't easily see why it doesn't work
2. the label question, I don't think the pattern considered this application, but the fix (`display: inline`) had to be pretty specific to override vanilla, perhaps it should be BEMed, but I think there is a larger issue with the instepping you are seeing, which I do not... can we look together tomorrow?

File: brand/canonical-logo/index.md (hunk 2 above)
Comment (pmahnke) [reply brand/canonical-logo/index.md]: fixed

File: _includes/side-nav-scopes.html (hunk 3 above)
Comment (pmahnke) [reply _includes/side-nav-scopes.html]: removed

File: _sass/_scratch.scss (hunk 1 above)
Comment (pmahnke) [reply _sass/_scratch.scss]: also, I need help getting the filters to work.  they were no longer working when I got here and are certainly not right now.

Comment (pmahnke) [reply _sass/_scratch.scss]: Oh...
3. on /brand, there are no sub-pages, so the whole accordion is open... which I agree is a bit odd, but that's what Yaili asked for.

**Comment by pmahnke** [comment]
ok @nottrobin ready for another review:

- I removed the accordion from /download 
- made the assets static with a yaml file
- uploaded all the assets to the asset server
- removed as much leftover js and css as I could find

File: _includes/header.html [hunk 4]
```diff
@@ -1,38 +1,49 @@
...
-      <input class="search__field" type="search" name="search" id="search" placeholder="Search design">
-
-      <input type="hidden" name="domain" value="docs.ubuntu.com/maas">
+<header id="navigation" class="p-navigation">
+  <div class="row">
+    <div class="p-navigation__banner">
+      <div class="p-navigation__logo">
+        <a class="p-navigation__link" href="/">
+          <svg xmlns="http://www.w3.org/2000/svg" height="27px" width="172px" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 172 27"><!-- Generator: Sketch 46.2 (44496) - http://www.bohemiancoding.com/sketch -->
+ <title>Artboard</title>
```
Comment (nottrobin) [inline _includes/header.html]: indentation

**Pull request merged**